app/build.gradle

```diff
@@ -205,6 +219,9 @@ dependencies {
     androidTestImplementation "com.nhaarman:mockito-kotlin-kt1.1:1.5.0"
     androidTestImplementation 'com.github.fabioCollini:DaggerMock:0.8.4'
     androidTestImplementation 'com.linkedin.testbutler:test-butler-library:1.3.1'
+
+    androidTestUtil 'com.android.support.test:orchestrator:1.0.2-alpha1'
```

Comment:
@stuartsoft Thoughts on if we should be pulling in Test Orchestrator until the issue linked at the bottom of the cited blog post is resolved? Unrelated, I'm just now learning about Test Orchestrator...looks cool!